filters/oscillatory_hallucination/README.md

```diff
@@ -0,0 +1,32 @@
+## Oscillatory Hallucinations Filter
+
+Author: Vikas Raunak (viraunak@microsoft.com)
+
+## What type of a filter is this?
+This filter is designed to operate in text generation systems' outputs, with the purpose of extracting oscillatory hallucinations. Oscillatory hallucinations are one class of hallucinations characterized by repeating bigram structure in the output [1]. Typically, these behaviors are observed in models trained on noisy corpora.
+
```

Comment:
Add Robustness Evaluation section

Reply:
For this filter, since the behavior caught is on model's output (rather than observing model's performance on filtered inputs), robustness evaluation isn't readily amenable to this filter. So, skipping this section as of now.